src/components/RadioButton/index.tsx

```diff
@@ -1,35 +1,25 @@
-import { MilkListProps } from 'types/common/main';
 import Form from 'react-bootstrap/Form';
 import React from 'react';
 
 interface RadioButton {
-  milkList: MilkListProps;
-  milk: string;
-  setMilk: (milk: string) => void;
+  type: 'radio' | 'checkbox';
+  id: string;
+  label: string;
+  handlechange: (value: string) => void;
+  check: boolean;
 }
 
-const CustomizeRadioButton = ({ milkList, milk, setMilk }: RadioButton) => {
+const CustomizeRadioButton = ({ type, id, label, handlechange, check }: RadioButton) => {
   return (
-    <>
-      {Object.keys(milkList).map((key) => {
-        const milkOption = milkList[key];
-        return (
-          <div key={key}>
-            <Form.Check
-              type={'radio'}
-              id={`milk-${key}`}
-              label={milkOption.name}
-              value={milkOption.name}
-              onChange={(e) => {
-                setMilk(e.target.value);
-              }}
-              checked={milk === milkOption.name}
-            />
-          </div>
-        );
-      })}
-    </>
+    <div key={id}>
```

Comment:
You can remove `div` since Form.Check is standalone.
Otherwise, you are wrapping div with two keys

Reply:
![image](https://user-images.githubusercontent.com/10414451/228114249-17318f74-22c4-4281-a467-09873b047235.png)

Reply:
![image](https://user-images.githubusercontent.com/10414451/228114310-1a54bd83-1a1c-4dc4-923b-465ee7a0f74d.png)

Reply:
Too much nesting